### DISCUSSION TITLE
Theme settings for pop_up path autocompletion

### DISCUSSION THREAD

**Discussion by psychemedia** [jupyterlab/jupyterlab#16725] [General]
Where are the colour settings for the pop-up path completion widget?

I am finding red on dark-blue is largely unreadable:

<img width="691" alt="image" src="https://github.com/user-attachments/assets/7833c0e4-0c5d-4fc7-ae48-68ce8f290b63">

**Comment by krassowski** [comment]
This is very poor contrast indeed. Are you using a custom theme which changes colour, or a browser extension?

This is how it looks in plain JupyterLab:

![image](https://github.com/user-attachments/assets/c05e0603-a802-4606-8204-46759ec8a024)

Snapshot taken directly from https://github.com/jupyterlab/jupyterlab/tree/main/galata/test/jupyterlab/completer.test.ts-snapshots

**Comment by krassowski** [comment]
By default the active option is `white`:

https://github.com/jupyterlab/jupyterlab/blob/f7ca6ad67e178995a70883ae284ecf5a7efd3d85/packages/completer/style/base.css#L123-L131

**Comment by psychemedia** [comment]
I'm using JupyterLab Light theme (I tried switching in and out of various other themes to see if that helped reset things, but it didn't).

There are various extensions installed but nothing that explicitly changes the theme settings as far as I know.

**Reply by krassowski** [reply]
Can you open use developer tools in browser on the active `.jp-Completer-item` and show what styles are active?

**Comment by psychemedia** [comment]
This maybe?

<img width="658" alt="image" src="https://github.com/user-attachments/assets/55972364-f0ab-436b-9e92-5b3ea181f30a">

**Reply by psychemedia** [reply]
also
<img width="252" alt="image" src="https://github.com/user-attachments/assets/4824ed00-9272-4e4e-a664-466561703bae">

**Reply by krassowski** [reply]
It looks like bootstrap is overriding it. Any extension or notebook output that is pulling/embedding bootstrap?

**Reply by psychemedia** [reply]
Ah, ok...there's quite a lot of extensions installed. Would that be something I could detect from network history?

**Reply by psychemedia** [reply]
<img width="1267" alt="image" src="https://github.com/user-attachments/assets/ebbf87e3-a46a-4645-b89f-131c3ed260b1">

**Reply by psychemedia** [reply]
> It looks like bootstrap is overriding it

So is that from your reading of the `--bs-...` element?

**Reply by krassowski** [reply]
Well this initiatior only tells us that it was loaded in an iframe. Do you have [jupysql-plugin](https://github.com/ploomber/jupysql-plugin) installed? It is known to break Jupyter styles:
- https://github.com/ploomber/jupysql-plugin/issues/109
- https://github.com/ploomber/jupysql-plugin/issues/101

**Reply by krassowski** [reply]
Yes, `--bs-code-color` is a bootstrap variable 100%

**Reply by psychemedia** [reply]
Yep,... I just started working through the extensions and found https://github.com/search?q=repo%3Aploomber%2Fjupysql-plugin%20bootstrap&type=code

I'm slowly filing issues with them about various sorts of brokeness that arise from that extension!
